packages/reports/src/Components/filter-component.tsx

```diff
@@ -58,17 +67,7 @@ const FilterComponent = ({
     );
 };
 
-FilterComponent.propTypes = {
-    action_type: PropTypes.string,
-    date_from: PropTypes.number,
-    date_to: PropTypes.number,
-    filtered_date_range: PropTypes.object,
-    handleDateChange: PropTypes.func,
-    handleFilterChange: PropTypes.func,
-    suffix_icon: PropTypes.string,
-};
-
-export default connect(({ modules }) => ({
+export default connect(({ modules }: any) => ({
```

Comment:
```suggestion
export default connect(({ modules }: RootStore) => ({
```

Reply:
Declare RootStore in the top of the file: `import RootStore from '../Stores/index';`

Reply:
Please checkout this as well https://github.com/mahdiyeh-deriv/deriv-app/pull/37/files#r1069147497

Reply:
Will change this 🙏🏼 to `RootStore`  thanks @akmal-deriv @niloofar-deriv

Reply:
@aizad-deriv please change the rootstore type to the Farzin's suggestion
![image](https://user-images.githubusercontent.com/82078941/214766667-2c99b45f-2d95-45ca-8bc9-7a0c921bb7ee.png)

Reply:
@mahdiyeh-deriv there's a PR regarding the root store [here](https://github.com/mahdiyeh-deriv/deriv-app/pull/40) by Sergei which has not yet been merged.

Reply:
@mahdiyeh-deriv I believe this is already done by @sergei-deriv in https://github.com/mahdiyeh-deriv/deriv-app/pull/40, And I think he has also done it in another PR 😅
Can we merge that PR if there is no concern about it so others can update their branches and use the `TRootStore`?

cc: @aizad-deriv @niloofar-deriv

Reply:
@farzin-deriv that PR as circle-ci issue, I'v asked sergei to fix that an I will merge it to the current branch we have for the reports then we can you it for reports migration. About using this changes in other packages @niloofar-deriv could help more.

Reply:
Yes I meant for `@deriv/reports` TS migration, Awesome, Thank you @mahdiyeh-deriv 🌹

Reply:
sure will help to changing them